.github/actions/upload-to-docker-hub/action.yml

```
@@ -0,0 +1,76 @@
+      shell: bash
+      run: |
+        mkdir -p /tmp/digests/${{ inputs.tag }}
+        echo "${{ steps.build.outputs.digest }}" > "/tmp/digests/${{ inputs.tag }}/${{ env.PLATFORM_DIR }}-digest.txt"
+
+    - name: Upload digest 📦
+      uses: actions/upload-artifact@v4
+      with:
+        name: docker-digest-${{ inputs.tag }}-${{ env.PLATFORM_DIR }}
+        path: /tmp/digests/${{ inputs.tag }}/${{ env.PLATFORM_DIR }}-digest.txt
+        retention-days: 1
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)